types/index.d.ts

```diff
@@ -55,7 +55,7 @@ export = TerserPlugin;
 /**
  * @typedef {Object} PredefinedOptions
  * @property {boolean} [module]
- * @property {any} [ecma]
+ * @property {5 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020 | number | string} [ecma]
```

Comment:
Webpack supports 2021/2022 too

Reply:
Updated with missing `2021 | 2022`

Reply:
Actually, I changed to use `TerserECMA` type